cohortextractor/query_utils.py

```diff
@@ -26,8 +26,7 @@ def get_column_definitions(cohort_class):
     for name, value in variables:
         if name.startswith("_") or name in ignored_names:
             continue
-        if not isinstance(value, Value):
-            raise ValueError(f"Cohort variable '{name}' is not a Value")
+        assert isinstance(value, Value), f"Cohort variable '{name}' is not a Value"
```

Comment:
This is a great case to raise. We should aim to make type errors like this literally impossible in the improved DSL.

Reply:
Yeah, I think agree with Becky here: this is a user error and therefore warrants an exception rather than an assertion. I think the fix for the coverage issue here is to add a tiny test case which triggers this condition.